native/src/components/BottomActionsSheet.tsx

```diff
@@ -22,6 +23,17 @@ const BottomActionsSheet: React.FC<BottomActionsSheetProps> = ({
 }: BottomActionsSheetProps): ReactElement | null => {
   const renderHandle = useCallback(props => <BottomSheetHandler title={title} {...props} />, [title])
 
+  const scrollRef = useRef<ScrollView>(null)
+
+  useEffect(() => {
```

Comment:
When does this happen?

Reply:
When the title changed so when the poi changed in Detail view.